app/konnect/analytics/use-cases.md

```diff
@@ -0,0 +1,190 @@
+Set the following options in the UI:
+
+* **Name**: Daily Accounts API Usage (last 30 days)
+* **Chart type**: Line chart
+* **Date/Time**: Last 30 days
+* **Metric**: Request count
+* **Group by**: Service
+* **Then by**: None
+
+Add a filter for the Account API. Click on **Add Filter**, then set the following options:
+
+* **Filter by**: Service
+* **Operator**: In
+* **Filter value**: Accounts API (or your own API name)
+
+![Daily Accounts API Usage (last 30 days)](/assets/images/docs/konnect/custom-reports/total-usage-accounts-api-30.png)
+
+Now that you know what day or days this spike happened on, let's dig a little deeper. 
+Edit the report to switch the metric from **Request Count** to **Requests per Minute**.
+
+Your configuration should now look like this, with the filter unchanged:
+
+* **Date/Time**: Last 30 days
+* **Metric**: Requests Per Minute
+* **Group by**: Service
+* **Then by**: None
+
+![Daily Accounts API Usage (last 30 days)](/assets/images/docs/konnect/custom-reports/total-api-usage-by-application-minute.png)
+
+These reports can provide your stakeholders with the answers to questions like:
+
+* What does average traffic over time look like for this service?
+* Are there any anomalies, such as spikes or drops in traffic?
+* If there are anomalies, do they repeat? Are there any patterns in the data?
+* If there's an issue with API traffic, has it been resolved or is it ongoing?
+
+#### Comparing daily vs per minute traffic
+
+Where a daily report can help stakeholders understand overall demand for the account API, the per minute report provides a more detailed picture. 
+By combining the two reports, you can provide a lot of insight into the usage and performance of your API. 
+
+For example, if the Accounts API is seeing a high number of requests per day and per minute, the organization may need to explore upgrading the API to handle the load. 
+Or, you may need to explore the current monetization policy to take advantage of the traffic.
+
+If the Accounts API is seeing a low amount traffic per day, but the per minute report shows normal traffic, you could be dealing with a variety of situations: 
+
+* Activity: The API is being used heavily during a specific part of the day. 
+* Security: Bots, or scripts, could be attempting to misuse your API. 
+* Stability: If the API is experiencing technical issues, such as slow response times or errors, could be resulting in a large number of retries that spike the requests per minute. 
+
+### Total API usage by {{site.konnect_saas}} application over the last 30 days
+
+You can track requests, you can do it per minute, per day, and you can even differentiate it across different APIs. However, because {{site.konnect_saas}} allows applications to consume services, you need to be able to answer what applications drive the most traffic to your APIs. Applications can be anything, public web applications, mobile applications, internal services that consume APIs.
```

Comment:
```suggestion
You can track requests per minute, per day, and even differentiate it across different APIs. However, because {{site.konnect_saas}} allows applications to consume services, you must be able to explain which applications drive the most traffic to your APIs. Applications can be anything: public web applications, mobile applications, and internal services that consume APIs.
```